airflow/providers/google/cloud/hooks/vertex_ai/generative_model.py

```diff
@@ -59,11 +61,23 @@ def get_generative_model(self, pretrained_model: str) -> GenerativeModel:
         model = GenerativeModel(pretrained_model)
         return model
 
+    @deprecated(
+        reason=(
+            "Please include `Part` objects as part of `contents` parameter in `airflow.providers.google.cloud.hooks.generative_model.GenerativeModelHook.generative_model_generate_content`"
```

Comment:
@CYarros10 could you please update a reason for the deprecation message and add a date when it will be removed, for example:
`
The “given” parameter is deprecated and will be removed after dd.mm.yyyy. Please use “this” parameter instead.
`
By default it is a 6 months.

Reply:
Done!